app/process/watcher/crypto-transfer/crypto-transfer.go

```diff
@@ -1,3 +1,90 @@
 package crypto_transfer
 
-// crypto transfer watcher
+import (
+	"encoding/json"
+	"fmt"
+	hederasdk "github.com/hashgraph/hedera-sdk-go"
+	"github.com/limechain/hedera-eth-bridge-validator/app/process/model/essential"
+	"github.com/limechain/hedera-eth-bridge-validator/app/process/model/transaction"
+	"github.com/limechain/hedera-eth-bridge-validator/config"
+	"github.com/limechain/hedera-watcher-sdk/queue"
+	"github.com/limechain/hedera-watcher-sdk/types"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strconv"
+	"time"
+)
+
+type CryptoTransferWatcher struct {
+	Account hederasdk.AccountID
+}
+
+func (ctw CryptoTransferWatcher) Watch(queue *queue.Queue) {
+	go beginWatching(ctw.Account, queue)
+}
+
+func getTransactionsFor(account hederasdk.AccountID, lastProcessedTimestamp string) (*transaction.Transactions, error) {
+	// TODO: Get last processed Tx timestamp
+	address := fmt.Sprintf("%s%s", config.LoadConfig().Hedera.MirrorNode.ApiAddress, "transactions")
+	accountLink := fmt.Sprintf("%s?account.id=%s&type=credit&result=success&timestamp=gt:%s&order=asc",
+		address,
+		account.String(),
+		lastProcessedTimestamp)
+
+	response, e := http.Get(accountLink)
```

Comment:
Let's extract the http calls to a separate client, e.g.
`httpClient.GetTransactionsByAccountIdAndTimestamp`